src/Tizen.NUI/src/internal/Model3dView.cs

```diff
@@ -66,116 +57,16 @@ protected override void Dispose(DisposeTypes type)
             base.Dispose(type);
         }
 
-        /// <since_tizen> 3 </since_tizen>
-        public new class Property : global::System.IDisposable
+        internal new class Property
```

Comment:
please make this HiddenAPI (not internal) ; please add [EditorBrowsable], [Obsolete] attributes like others done.